### DISCUSSION TITLE
Possibility of translation for Bestiary

### DISCUSSION THREAD

**Discussion by Houbsi** [Obsidian-TTRPG-Community/fantasy-statblocks#250] [Feature Requests]
Hi there,

i know a bit of coding. But can't do the heavy lifting at the moment for the outer issue. But how hard would it be to make the bestiary translated into the official translations?

I'm nativ german speaking and my only solution atm is to fork the whole repo and overwrite this file: https://github.com/valentine195/obsidian-5e-statblocks/blob/master/src/data/srd-bestiary.ts

Maybe a hint or some help is appreciated so that others can also profit from this.

**Comment by sigrunixia** [comment]
My apologies for the belated reply, @Houbsi. I'm helping Javalent get caught up on these.

If you are still interested in this, all you would do is take that file, apply the official translations to it, rename it, and PR it back. We would do the extra work on our side to make it available as an option in Obsidian.

**Comment by Houbsi** [comment]
> My apologies for the belated reply, @Houbsi. I'm helping Javalent get caught up on these.
> 
> If you are still interested in this, all you would do is take that file, apply the official translations to it, rename it, and PR it back. We would do the extra work on our side to make it available as an option in Obsidian.

Will do :) Don't know when i'll have the time currently but expect something in the next few months.

**Comment by sigrunixia** [comment]
Awesome! I'll keep this pinned so the stale bot wont eat it.
